distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -264,4 +432,63 @@ private void showServerReply(FTPClient ftpClient) {
             }
         }
     }
-}
+
+    /**
+     * Sets the clientAuth configuration in catalog-ftp feature.
+     *
+     * @throws Exception
+     */
+    private void setClientAuthConfiguration(String clientAuth) throws Exception {
+        Configuration config =
+                getAdminConfig().getConfiguration("ddf.catalog.ftp.FtpServerStarter");
+        config.setBundleLocation("mvn:ddf.catalog/ftp/" + System.getProperty("ddf.version"));
+        Dictionary properties = new Hashtable<>();
+
+        properties.put("port", FTP_DEFAULT_PORT);
```

Comment:
@tbatie This ticket is out here right now to allow dynamic ports and should be merged before this ticket so it can be changed here.
https://github.com/codice/ddf/pull/1128